NDTensors/src/lib/Unwrap/src/unwraptype.jl

```diff
@@ -0,0 +1,19 @@
+using NDTensors.TypeParameterAccessors: IsWrappedArray, parenttype
+using SimpleTraits: SimpleTraits, @traitfn
+## These functions will be used in place of unwrap_type but will be
+## call indirectly through the expose function.
+@traitfn function unwrap_type(
```

Comment:
Probably we should call this `unwrap_array_type` since it designed primarily for arrays.

Reply:
Sounds good, I changed this across the code base